README.md

```diff
@@ -1,14 +1,15 @@
-Egg Chess v1.1 (010910) by Anders Lindstr�m
-============================================
+# Egg Chess v1.1 (010910) by Anders Lindstr�m
 
-1. Intro
+![EggChess screenshot](EggChess.png "EggChess")
+
+### 1. Intro
 
 This is free software (GNU GPL), read COPYING for details.
```

Comment:
...and remove that one, too. There's no COPYING file currently, but we can add a LICENSE file. Anyway, no need to mention it in the readme.